extension/content.js

```diff
@@ -194,6 +215,8 @@ document.addEventListener('DOMContentLoaded', () => {
 	}
 
 	if (isRepo) {
+		const isRepoTree = window.location.href.split('/').length === 5 || window.location.href.split('/')[5] === 'tree';
```

Comment:
The _real_ root is `location.pathname === `/${ownerName}/${repoName}/``. That's making sure you're at `/sindresorhus/refined-github/` for example. If you want to support `/sindresorhus/refined-github/tree/master` you have to support all trees since it's possible that the default branch is not master (e.g. having `gh-pages` as the default for a jekyll site or something).